frontend/Container.js

```diff
@@ -37,14 +43,42 @@ class Container extends React.Component {
     },
     extraTabs: {[key: string]: () => React$Element},
   };
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isVertical: (window.innerWidth < 500),
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', (e) => this.handleResize(e), false);
```

Comment:
Please throttle this event listener. Something like once every 100ms should be fine.